behat.yml.dist

```diff
@@ -21,6 +21,15 @@ default:
                 - Sylius\Bundle\WebBundle\Behat\WebContext
                 - Sylius\Bundle\MoneyBundle\Behat\MoneyContext
                 - Sylius\Bundle\ChannelBundle\Behat\ChannelContext
+                - Sylius\Behat\Context\Setup\UserContext:
```

Comment:
Rebase with master, put this suite in `etc/behat/suites/ui-user.yml` and link to that file in `etc/behat/suites.yml` :)